build/setup.d/21-install-td-agent-packages.sh

```diff
@@ -0,0 +1,14 @@
+# Ensure that /etc/init.d/td-agent is removed
+rm /etc/init.d/td-agent
+
+# Ensure that directories are writable
+chmod -R 1777 /tmp/
```

Comment:
Why is this needed?

Reply:
I can't remember the exact reason. I will revert this change and see if we still require it.

Reply:
This doesn't appear to be necessary.